var/spack/repos/builtin/packages/lcms/package.py

```diff
@@ -35,6 +37,27 @@ def url_for_version(self, version):
     depends_on("libtiff")
     depends_on("zlib-api")
 
+    build_system("autotools", "msbuild")
+
     @property
     def libs(self):
         return find_libraries("liblcms2", root=self.prefix, recursive=True)
```

Comment:
(minor) Generally I assume `runtime=False` should be on each `.libs` implementation (which suggests to me perhaps we should default it to `False`, but something to be addressed later.

Reply:
Agreed we should default to it, but there are enough special cases that we can't blindly implement it for all `.libs`. We'd probably be right for 99% of cases though. Basically things that are intended to be plugins should find the runtime, and a few other cases (Python extensions, etc)